src/ui/molecules/stepper/stepper.scss

```diff
@@ -81,17 +81,17 @@ $colors: (
     box-shadow: hsl(206 22% 7% / 35%) 0 10px 38px -10px, hsl(206 22% 7% / 20%) 0 10px 20px -15px;
   }
 
+  &.error {
+    @include with-theme() {
```

Comment:
Also, I hadn't seen it before but I think you can remove `@include with-theme()` from the `error` because it is already present in `.okp4-stepper-body`. 
And since the `error` is from .`okp4-stepper-body` it can benefit from it.
We could move it just below `.okp4-stepper-body` by the way, because it is usually just below the classname.